src/components/bottom-tabs/BottomTabsScreen.types.ts

```diff
@@ -95,24 +95,40 @@ export interface BottomTabsScreenProps {
 
   // #region General
   /**
-   * @summary Title of the tab screen, displayed in the tab bar.
+   * @summary Title of the tab screen, displayed in the tab bar item.
    *
    * @platform android, ios
    */
   title?: string;
   /**
    * @summary Specifies content of tab bar item badge.
    *
-   * @todo Describe prop behavior on Android.
    * On iOS, badge is displayed as regular string.
    *
+   * On Android the value is interpreted as follows:
+   * - displays the value as a number, when the string can be parsed to integer
+   * - displays "small dot" badge, when the string is empty
+   * - displays the value as a text, otherwise
```

Comment:
I'd change the order here
```suggestion
   * - if the string can be parsed to integer, displays the value as a number
   * - otherwise if the string is empty, displays "small dot" badge
   * - otherwise, displays the value as a text 
```

Reply:
changed in: https://github.com/software-mansion/react-native-screens/pull/3070/commits/707d15f1ee3da6788414d3a178a3aa810f517349